pkg/controller/nodeipam/ipam/multinetwork_cloud_cidr_allocator.go

```diff
@@ -26,13 +30,28 @@ func (ca *cloudCIDRAllocator) performMultiNetworkCIDRAllocation(node *v1.Node, i
 	if err != nil {
 		return nil, fmt.Errorf("node=%s error fetching networks: %v", node.Name, err)
 	}
+
+	// get networks from Node's network-status annotation
+	nodeUpNet, err := getUpNetworks(node)
+
+	// statusNetworks is map of networks that are Ready and status==UP
+	statusNetworks := make(map[string]struct{})
```

Comment:
call this `readyNetworks`?

Reply:
I renamed readyAndUpNetworks. Its a bit more verbose, but readyNetworks would've specifically referred to the network object's ready condition.